src/app/clients/clients.service.ts

```diff
@@ -1,347 +1,175 @@
+/** Angular Imports */
```

Comment:
You are not supposed to change this file completely. Please only keep the changes relevant to your component specified in the issue. Do not change for all the components. Refer https://github.com/openMF/web-app/pull/318 to avoid conflicts.